File: P1/DAA001.java
import java.util.Scanner;

public class DAA001 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        
        int n = in.nextInt();
        int total = 0;
        for(int i = 0; i < n; i++) {
            if (in.nextInt() == 42) total++;
        }
        System.out.println(total);
        in.close();   
    }
}

File: P1/DAA001.c
#include <stdio.h>

int main() {
    int n, k;
    scanf("%d", &n);
    int total = 0;
    for (int i = 0; i < n; i++)
    {
        scanf("%d", &k);
        if (k == 42) total++;
    }
    printf("%d\n", total);
    return 0;
}